ast-visual-studio-extension/CxExtension/Toolbar/BranchesCombobox.cs

```diff
@@ -99,12 +99,15 @@ public void OnChangeBranch(object sender, SelectionChangedEventArgs e)
         {
             if (!(sender is ComboBox branchesCombo) || branchesCombo.SelectedItem == null || branchesCombo.SelectedIndex == -1) return;
 
+            ResetFilteringState(branchesCombo.SelectedItem as ComboBoxItem);
+
+            string selectedBranch = (branchesCombo.SelectedItem as ComboBoxItem).Content as string;
+
             cxToolbar.EnableCombos(false);
             cxToolbar.ScansCombo.Text = string.IsNullOrEmpty(CxToolbar.currentScanId) ? CxConstants.TOOLBAR_LOADING_SCANS : CxToolbar.currentScanId;
             cxToolbar.ResultsTreePanel.ClearAll();
 
-            string selectedBranch = (branchesCombo.SelectedItem as ComboBoxItem).Content as string;
-            string projectId = ((cxToolbar.ProjectsCombo.SelectedItem as ComboBoxItem).Tag as Project).Id;
+            string projectId = ((cxToolbar.ProjectsCombo.SelectedItem as ComboBoxItem).Tag as ast_visual_studio_extension.CxWrapper.Models.Project).Id;
```

Comment:
change it - ast_visual_studio_extension.CxWrapper.Models in the using (add to using)